cli/Dockerfile

```
@@ -1,15 +1,20 @@
+RUN apk add --no-cache bash-completion
 WORKDIR /usr/app
 RUN chown node:node /usr/app
 RUN mkdir ./opfab-cli
 COPY src ./opfab-cli/
 COPY entryPoint.sh /
+COPY loadCompletion.sh /
+RUN mkdir /opfab
+RUN chmod 777 /opfab
+COPY bashrcForDocker /opfab/.bashrc
 RUN chmod +x /entryPoint.sh
 WORKDIR ./opfab-cli
```

Hadolint

[error] Use absolute WORKDIR

(DL3000)